app/main.py

```diff
@@ -4,0 +1,73 @@
+import json
+from app.shop import Shop
+from app.customer import Customer
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as file:
+        config = json.load(file)
+
+    fuel_price = config["FUEL_PRICE"]
+    customers = []
+    shops = []
+
+    for shop_data in config["shops"]:
+        shops.append(Shop(**shop_data))
+
+    for customer_data in config["customers"]:
+        car = Car(**customer_data.pop("car"))
+        customers.append(Customer(**customer_data, car=car))
+
+    for customer in customers:
+        print(f"{customer.name} has {customer.money} dollars")
+        best_shop = None
+        best_cost = float("inf")
+
+        for shop in shops:
+            trip_cost, distance = shop.calculate_trip_cost(
+                customer.location,
+                fuel_price,
+                customer.car.fuel_consumption
+            )
+
+            product_cost, _ = shop.purchase(
+                customer.name, customer.product_cart
+            )
+            total_cost = trip_cost + product_cost
+
+            print(f"{customer.name}'s trip to "
+                  f"the {shop.name} costs {round(total_cost, 2)}")
```

Comment:
Ensure that the total cost is rounded to two decimal places when printed. This is important for maintaining consistency in the output format.